front/app/components/UI/FullscreenModal/index.tsx

```diff
@@ -91,6 +88,7 @@ interface Props extends InputProps {
 
 interface State {
   windowHeight: number;
+  windowWidth: number;
 }
 
 class FullscreenModal extends PureComponent<Props, State> {
```

Comment:
Should we add making this into a functional component to our code cleanup ticket? :)

Reply:
Definitely! My fingers were itching but trying to keep PRs focused. :) I already had a [ticket](https://www.notion.so/govocal/Further-clean-up-of-FullscreenModal-1439663b7b2680e98c52e9478f62e90e?pvs=4) for it.